lib/engine/round/stock.rb

```diff
@@ -173,6 +173,12 @@ def action_finalized(_action)
         end
       end
 
+      def par_prices(corporation)
```

Comment:
as discussed we may remove this not sure yet

Reply:
I think there's quite a few other games that vary par prices by corporation type and by game phase, so i'd suggest it'll be needed eventually.

Reply:
it may make sense then for the corporation to have it's own par prices, i already have a hack called min_value that could benefit from this. i'd suggest moving that info there

Reply:
I'll fix this one tomorrow and the other remaining issue